From c6b73564f334f2424f015cfd436e13e2a0db5c46 Mon Sep 17 00:00:00 2001
From: Konrad Weihmann <kweihmann@outlook.com>
Date: Sun, 27 Feb 2022 10:05:17 +0100
Subject: [PATCH] setup.py: use setuptools instead of distutils

as distutils will be deprecated in python 3.12 and already produces
a warning when run with python 3.10.x, it might be a good time to
switch to setuptools.
It offers the same interface, so results will be the same

Signed-off-by: Konrad Weihmann <kweihmann@outlook.com>
Upstream-Status: Submitted
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index df00fcb..8882fb7 100644
--- a/setup.py
+++ b/setup.py
@@ -1,4 +1,4 @@
-from distutils.core import setup
+from setuptools import setup
 
 setup(
     name='systemdunitparser',
-- 
2.25.1

